backend/googlephotos/googlephotos.go

```diff
@@ -132,15 +132,26 @@ you want to read the media.`,
 			Default:  2000,
 			Help:     `Year limits the photos to be downloaded to those which are uploaded after the given year`,
 			Advanced: true,
+		}, {
+			Name:    "include_archived",
+			Default: false,
+			Help: `Also download archived media.
+
+By default rclone do not request archived media. Thus when syncing,
```

Comment:
Apologies for this, but do you mind if I re-write this help a bit?

```
By default rclone does not request archived media. Thus, when syncing,
archived media is not visible in directory listings or transferred.

Note that media in albums is always visible and synced, no matter
their archive status.

With this flag, archived media are always visible in directory
listings and transferred.

Without this flag, archived media will not be visible in directory
listings and won't be transferred.
```

Reply:
Nice, I included your suggestion in my pull request.